gpu/features/src/fpfh.cu

```diff
@@ -39,19 +39,11 @@
 #include "pcl/gpu/utils/safe_call.hpp"
 #include "pcl/gpu/utils/device/warp.hpp"
 #include "pcl/gpu/utils/device/functional.hpp"
-#include "pcl/gpu/utils/device/funcattrib.hpp"
-#include "pcl/gpu/utils/timers_cuda.hpp"
 #include "pcl/gpu/features/device/pair_features.hpp"
 
-#include <iostream>
-
 using namespace pcl::gpu;
 using namespace pcl::device;
 
-#ifndef M_PI
-#define M_PI 3.14159265358979323846f
-#endif
```

Comment:
`M_PI` is used in this file further down, and I assume there is a reason why these lines have been added in the past, so I would rather keep them. Or is there a reason why you removed them? Did they cause any problem for you?

Reply:
Seems to be required on windows - alternatively we could include #include <pcl/pcl_macros.h> as it has the definition as well. Same for vfh.cu.

Reply:
Yeah, and gpu aren't build on windows CI, so we have to verify explicitly.

Reply:
I would prefer putting the three lines back